packages/ui/src/hooks/useDisplayError.tsx

```diff
@@ -49,6 +51,30 @@ export const useDisplayError = () => {
     )
   }
 
+  if (!canFindMultiProxyFromUrl) {
+    return (
+      <CenterStyled>
+        <ErrorMessageStyled>
+          <ErrorOutlineIcon size={64} />
+          <div data-cy="label-linked-address-not-found">
+            The linked address can't be found in your accounts or watched accounts.
+          </div>
+          <Button
+            data-cy="button-reset-linked-address"
+            onClick={() =>
+              setSearchParams((prev) => {
+                prev.delete('address')
+                return prev
+              })
+            }
+          >
+            Reset
```

Comment:
I agree that the "Reset" looks weird here, maybe "Back to home" will be more understandable in terms of UI.

Reply:
Yes, I need to create an issue for this. Agreed that Back to Home would already be better.